include/igl/embree/EmbreeIntersector.h

```diff
@@ -176,6 +169,8 @@ namespace igl
       Triangle* triangles;
       bool initialized;
 
+      RTCDevice g_device;
```

Comment:
It is not a global variable, so I'd remove the `g_` prefix. (It'd be more correct to call it `device_handle` but that might be too verbose?).